apps/blockchain/test/blockchain/block_test.exs

```diff
@@ -9,40 +9,47 @@ defmodule Blockchain.BlockTest do
   alias EVM.MachineCode
   alias MerklePatriciaTree.Trie
 
-  define_common_tests("GenesisTests", [], fn _test_name, test_data ->
-    for {internal_test_name, test} <- test_data do
-      if Enum.member?(["test2", "test3"], internal_test_name) do
-        db = MerklePatriciaTree.Test.random_ets_db()
-
-        chain = %Chain{
-          genesis: %{
-            timestamp: test["timestamp"] |> maybe_hex,
-            parent_hash: test["parentHash"] |> maybe_hex,
-            extra_data: test["extraData"] |> maybe_hex,
-            gas_limit: test["gasLimit"] |> maybe_hex,
-            difficulty: test["difficulty"] |> maybe_hex,
-            author: test["coinbase"] |> maybe_hex,
-            seal: %{
-              mix_hash: test["mixHash"] |> maybe_hex,
-              nonce: test["nonce"] |> maybe_hex
-            }
-          },
-          accounts: get_test_accounts(test["alloc"])
-        }
+  test "eth common tests" do
+    common_tests = EthCommonTest.Helpers.test_files("GenesisTests")
+
+    for test_path <- common_tests do
+      test_data = EthCommonTest.Helpers.read_test_file(test_path)
+
+      for {internal_test_name, test} <- test_data do
+        # TODO: Enum.member? why?
```

Comment:
I think it's better to create GitHub issues instead of leaving `TODO` comments in the code. Also, `TODO` comments are credo warnings (currently, it's disabled)